server.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const express = require('express');
+const helmet = require('helmet');
+
+const server = express();
+
+const database = require('./data/database');
+
+const routes = require('./user/routes');
+const user = require('./user/User');
```

Comment:
We don't need to import the User schema here either. It's only being used inside `./user/routes.js`